### PR TITLE
fix conditional types in TL-B

### PR DESCRIPTION
**Pull request by mr-tron**
## Why is it important?

Docs is incorrect now.

### CODE REVIEW

**Comment by mr-tron** [comment]
https://github.com/ton-community/ton-docs/pull/113 duplicate

File: docs/learn/overviews/tl-b-language.mdx [hunk 1]
```diff
@@ -211,4 +211,4 @@ The TL-B grammar is described in the [TlbParser.bnf](https://github.com/andreypf
 - [A description of an older version of TL](https://core.telegram.org/mtproto/TL)
 - [block.tlb](https://github.com/newton-blockchain/ton/blob/master/crypto/block/block.tlb)
 
-_Thanks to the [Vudi](https://github.com/Vudi) and [cryshado](https://github.com/cryshado) for contributing to the community!_
\ No newline at end of file
+_Thanks to the [Vudi](https://github.com/Vudi) and [cryshado](https://github.com/cryshado) for contributing to the community!_
```
Comment (Rocksriram) [inline docs/learn/overviews/tl-b-language.mdx:214]: Withdraw my account

Comment (MOJTABAmj259) [reply docs/learn/overviews/tl-b-language.mdx:214]: > Withdraw my account

Comment (MOJTABAmj259) [reply docs/learn/overviews/tl-b-language.mdx:214]: 123456789

**Pull request closed without merging**